src/components/transactions/Warnings/ETHBorrowWarning.tsx

```diff
@@ -0,0 +1,31 @@
+import { ExclamationIcon } from '@heroicons/react/outline';
+import { Trans } from '@lingui/macro';
+import { Box, SvgIcon } from '@mui/material';
+import { ContentWithTooltip } from 'src/components/ContentWithTooltip';
+import { Link } from 'src/components/primitives/Link';
+
+export const ETHBorrowWarning = () => {
+  return (
+    <ContentWithTooltip
```

Comment:
I noticed a bug while testing this with the `ContentWithTooltip` component. Click events aren't prevented from being propagated, so if you click on the link to the in this tooltip, it will also navigate you to the reserve overview page (only on the markets page). I added an issue for this #1093

Reply:
Ended just fixing as part of this PR